doc-content/enterprise-only/admin-and-config/log-process-instance-con.adoc

```diff
@@ -0,0 +1,155 @@
+[id='log-process-instance-con']
+
+= Logging process instances
+
+Logging is a mechanism that allows you to store information about the execution of a process instance. You can log a process instance using a special event listener that listens to the {PROCESS_ENGINE} for any relevant events to be logged. Later, the information is stored separately from other non-log information that is stored either in the server built-in database (H2) or in a connected data source JPA or Hibernate.
+
+You can use the `jbpm-audit` module that provides an event listener and allows you to store the information directly in a database using JPA or Hibernate.
+
+[source,subs="attributes+"]
+----
+<dependency>
+	<groupId>org.jbpm</groupId>
+	<artifactId>jbpm-audit</artifactId>
+	<version>{PRODUCT_VERSION_LONG}-redhat-2</version>
+</dependency>
+----
```

Comment:
This kind of information cannot be really set when installing it on EAP or JWS. It can only be set when running a process engine inside a custom application. So I wouldn't add it into EAP or JWS install guides but probably to the guide which is about running engine as a part of a custom app.

Reply:
Right, this setup is already available in Kie server.

Reply:
Therefore, I would suggest to move this to another chapter, definitely somewhere where we have configuration of jBPM engine as a part of a custom app.

Reply:
@MarianMacik and @sutaakar, thanks for providing the inputs, I will be moving this section to process engine doc